toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -323,12 +326,10 @@ Options for configuring the kernel.
 
 Additional Linux kernel command line options to add to the image.
 
-If the partitions are customized, then the `grub.cfg` file will be reset to handle the
-new partition layout.
-So, any existing ExtraCommandLine value in the base image will be replaced.
-
-If the partitions are not customized, then the `ExtraCommandLine` value will be appended
-to the existing `grub.cfg` file.
+If [ResetBootLoaderType](#resetbootloadertype-string) is set to `"hard-reset"`, then the
+`ExtraCommandLine` value will be appended to the new `grub.cfg` file.
+Otherwise, the `ExtraCommandLine` value will be appended to the existing `grub.cfg`
```

Comment:
Is that an option?

Reply:
It is. I've rewrote this to hopefully make it clearer.